js/index.js

```diff
@@ -40,3 +40,71 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+const topNav = document.querySelectorAll('a'); 
+for(let i = 0; i < topNav.length; i++) {
+  topNav[i].textContent = siteContent['nav'][`nav-item-${i + 1}`]; 
+  topNav[i].style.color = 'green'; 
+}; 
```

Comment:
excellent way to automate the nav link text AND the text color at the same time.